source/_posts/2017/2017-01-20-4-emotional-reasons-I-quit-my-twitter.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: post
+title: "4 Emotional and Honest Reasons I quit my Twitter"
+perex: '''
+    Nowadays we tend to live busy life with all those social networks, notifications and pings. So busy, we don't have emotional space to just evaluate "How do I like that?". In my life, this creates cycles, where I keep doing over and over the same thing without any progress. After few months feeling bad with Twitter, I've finally decided to quit. Rather from technical point of view, I will focus on my feelings. Because that's what matters most.
```

Comment:
I know what you mean. This way it has better meaning of surprise in the end. Copy over grammar.